### PR TITLE
[WIP] MLOPS-384 Skip installation of rdkit in Windows

### PR DESCRIPTION
**Pull request by RufusGladiuz**
Added exception for rdkit import

### CODE REVIEW

File: omigami/plotting.py [hunk 1]
```diff
@@ -3,9 +3,20 @@
...
+import warnings
+
+try:
+    from rdkit import Chem
+    from rdkit.Chem import Draw
+    from rdkit.Chem.rdchem import Mol
+except ModuleNotFoundError:
+    import platform
+    if platform.system() == "Windows":
+        warnings.warn("You are missing the rdkit module, unfortunately rdkit cant be installed by using pip on Windows."
```
Comment (dianacarvalho1) [inline omigami/plotting.py:15]:
```suggestion
        warnings.warn("You are missing the rdkit module, unfortunately rdkit can't be installed by using pip on Windows."
```

**Review by dianacarvalho1 — COMMENTED** [review]
Thaaaanks!! Could you also add a warning to the ReadMe? And what if the user tries to run these functions? We should throw an error then no? I will test this on windows now to see how it is!

File: omigami/plotting.py [hunk 2]
```diff
@@ -48,37 +62,43 @@ def plot_molecule_structure_grid(
...
+            image = Draw.MolsToGridImage(
+                mol_render_list,
+                subImgSize=molecule_image_size,
+                legends=spectra_matches.compound_name.tolist(),
+                highlightBondLists=highlight_bonds,
+            )
+            return image
+        except NameError:
+
+            warnings.warn("You are missing the rdkit module, "
```
Comment (dianacarvalho1) [inline omigami/plotting.py:98]: I think this should be really a raise no? like it won't work either way

File: omigami/plotting.py [hunk 3]
```diff
@@ -122,15 +142,16 @@ def _get_bonds_to_highlight(molecule: Mol, substructure: Mol) -> List[int]:
...
         for atom in molecule.GetAtoms():
             atom.SetAtomMapNum(atom.GetIdx())
         return molecule
 
+
     # Original Source: https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y
     @staticmethod
     def plot_classyfire_result(
-        spectra_matches: pd.DataFrame, color="g"
+            spectra_matches: pd.DataFrame, color="g"
```
Comment (dianacarvalho1) [inline omigami/plotting.py:154]: doesn't this one need the try catch as well?

File: omigami/plotting.py [hunk 4]
```diff
@@ -155,7 +176,7 @@ def plot_classyfire_result(
     # Original Source: https://chemrxiv.org/engage/chemrxiv/article-details/60c74f58702a9ba8dc18bb6b
     @staticmethod
     def plot_NPclassifier_result(
-        spectra_matches: pd.DataFrame, color="g"
+            spectra_matches: pd.DataFrame, color="g"
```
Comment (dianacarvalho1) [inline omigami/plotting.py:179]: and this one too?

**Review by dianacarvalho1 — CHANGES_REQUESTED** [review]
just a few more changes

File: omigami/plotting.py (hunk 3 above)
Comment (RufusGladiuz) [reply omigami/plotting.py:154]: No, this one works without rdkit :)

File: omigami/plotting.py (hunk 4 above)
Comment (RufusGladiuz) [reply omigami/plotting.py:179]: Same here :)

**Review by dianacarvalho1 — APPROVED** [review]
👍🏼

**Pull request merged**